sw/otbn/code-snippets/randomness.s

```diff
@@ -111,11 +113,36 @@ test_urnd:
   /* Read the URND WSR again. */
   bn.wsrr w1, 0x2 /* URND */
 
-  /* Fail the test if the two numbers read from URND are equal. */
+  jal x0, error_checking
+
+/**
+ * This sobroutine performs the following verifications:
```

Comment:
Nit: s**u**broutine. Also, "verifications" isn't all that idiomatic here. Maybe "checks"?

Reply:
Thanks. Done!